components/ui/ModalCreation.vue

```diff
@@ -69,6 +82,41 @@ import CheckIcon from '~/assets/images/utils/right-arrow.svg'
 import Modal from '~/components/ui/Modal.vue'
 import Chips from '~/components/ui/Chips.vue'
 
+const messages = defineMessages({
+  createProjectModalDescription: {
```

Comment:
I don't think it really makes sense to prefix all properties with `createProjectModal`, these properties are only for developer to use in this file, they are never exported, only IDs are.

Something like
```suggestion
  modalDescription: {
```

and then below go like
```js
  nameLabel:
```